dlp/api/Snippets/InspectDataWithHotwordRule.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_inspect_hotword_rule]
+
+using System;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using static Google.Cloud.Dlp.V2.CustomInfoType.Types;
+
+public class InspectDataWithHotwordRule
+{
+    public static InspectContentResponse InspectDataHotwordRule(
+        string projectId,
+        string text,
+        string customRegex,
+        string hotwordRegex)
+    {
+        // Instantiate dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the content item.
+        var contentItem = new ContentItem
+        {
+            ByteItem = new ByteContentItem
+            {
+                Type = ByteContentItem.Types.BytesType.TextUtf8,
+                Data = Google.Protobuf.ByteString.CopyFromUtf8(text)
+            }
+        };
+
+        // Construct the info type.
+        var infoType = new InfoType { Name = "C_MRN" };
```

Comment:
I suggest parameterising the infoType name as well. That way the entire custom info type including its name will be controlled by the caller.

Reply:
Done.